viral/offline_sequence_detection.py

```
@@ -0,0 +1,567 @@
+from sklearn.model_selection import train_test_split
+
+HERE = Path(__file__).parent
+sys.path.append(str(HERE.parent))
+sys.path.append(str(HERE.parent.parent))
+
+from viral.constants import CACHE_PATH, TIFF_UMBRELLA
+from viral.models import (
+    Cached2pSession,
+    GrosmarkConfig,
+    BayesianDecodingConfig,
+    Cached2pSession,
```

Ruff (0.14.1)

Redefinition of unused `Cached2pSession` from line 18

Remove definition: `Cached2pSession`

(F811)